cleanrl/ppo.py

```diff
@@ -213,18 +213,18 @@ def get_action_and_value(self, x, action=None):
                     writer.add_scalar("charts/episodic_length", item["episode"]["l"], global_step)
                     break
```

Comment:
This part may need to be changed to 

```
            if "episode" in info:
                for item in info["episode"]["r"]:
                    print(f"global_step={global_step}, episodic_return={item}")
                    writer.add_scalar("charts/episodic_return", item, global_step)
                    break
                for item in info["episode"]["l"]:
                    writer.add_scalar("charts/episodic_length", item, global_step)
                    break
```

Reply:
To replicate the original behavior, you probably need something like (but hopefully better-looking than!):
```python
if "episode" in info:
    first_idx = info["_episode"].nonzero()[0][0]
    r = info["episode"]["r"][first_idx]
    l = info["episode"]["l"][first_idx]
    print(f"global_step={global_step}, episodic_return={r}")
    writer.add_scalar("charts/episodic_return", r, global_step)
    writer.add_scalar("charts/episodic_length", l, global_step)
```
There's no guarantee that the first index in "episode" won't just be a zero, need the mask to specify which one.

Alternatively, it might be better to track a running average using the deques built into the RecordEpisodeStatistics wrapper, though that would likely results in different performance graphs.